roles/rhscl-mariadb-apb-openshift/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+- include: dev.yml
+  when: _apb_plan_id == "dev"
+
+- include: prod.yml
+  when: _apb_plan_id == "prod"
+
+- name: set {{ service_name }} service state to {{ state }}
+  k8s_v1_service:
+    name: "{{ service_name }}"
+    namespace: '{{ namespace }}'
+    state: present
```

Comment:
This will never get deprovisioned. Need "{{ state }}"

Reply:
:sunglasses: